packages/core/src/plugins/asset.ts

```diff
@@ -108,18 +108,19 @@ export const pluginAsset = (): RsbuildPlugin => ({
 
       const { emitAssets } = config.output;
 
-      // svg
-      createAssetRule('svg', ['svg'], emitAssets);
-      // font
-      createAssetRule('font', FONT_EXTENSIONS, emitAssets);
-      // image
-      createAssetRule('image', IMAGE_EXTENSIONS, emitAssets);
+      // put the easy-to-hit and high-frequency rule behind (svg, image), which would be hit as early as possible and have better performance
```

Comment:
As we do not use top-level `oneof`, all top-level rules are matched and changing the rule order should have no effect on the build performance.

Reply:
maybe svg files will hit the rule eailer than before, isn't it?

Reply:
All rules are still to be matched...